index.html

```diff
@@ -0,0 +1,128 @@
+            src="./Components/Pages/Assets/icon-game.gif" alt="icon"></a>
+        <button class="navbar-toggler me-auto" type="button" data-bs-toggle="collapse"
+          data-bs-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false"
+          aria-label="Toggle navigation">
+          <span class="navbar-toggler-icon"></span>
+        </button>
+        <div class="collapse navbar-collapse" id="navbarSupportedContent">
+          <ul class="navbar-nav ms-auto mb-2 mb-lg-0">
+            <li class="nav-item">
+              <a class="nav-link active" aria-current="page"
+                href="./Components/Pages/About/Component/about.html">About</a>
+            </li>
+            <li class="nav-item">
+              <a class="nav-link disabled">Program</a>
+            </li>
+            <li class="nav-item">
+              <a class="nav-link disabled">Join</a>
+            </li>
+            <li class="nav-item">
+              <a class="nav-link disabled">Sponsor</a>
+            </li>
+            <li class="nav-item">
+              <a class="nav-link disabled">News</a>
+            </li>
+            <li class="nav-item mx-5 nav-content">
+              <a class="nav-link">CC Campain</a>
+            </li>
+          </ul>
+        </div>
+      </div>
+    </nav>
+    <div class="body-bg d-flex justify-content-center">
+      <div class="header">
+        <video class="d-flex justify-content-center" id="background-video" autoplay loop muted
+          poster="./Components/Pages/Assets/mkke-bg.png">
+          <source src="./Components/Pages/Assets/mkkefinal.mp4" type="video/mp4">
+        </video>
+        <div class="title d-flex justify-content-center flex-column">
+          <h4>"Hello! Sharing world"</h4>
+          <h1>RANKING OF VIDEO GAMES IN 2021</h1>
+          <div class="descruption">
+            <p>Games are more than just games. It's odd phrasing but it's the truth. They are meant to enrapture, encapture, pull in a player and audience through three methods: visuals, gameplay, and narrative. we are going to list best game ranking in the world with some details according to testimonies of players
+            </p>
+          </div>
+          <div class="prod-year">
+            <h3>
+              2019.10.15 (Monday) ~ 16(Tuesday)
+            </h3>
+            <p>@informations share by regiss mukubiza</p>
+          </div>
+        </div>
+      </div>
+    </div>
+  </header>
+  <div class="container-programs">
+    <div class="main-program-container container-fluid text-light programs-class">
+      <h1 class="text-center my-2">Main Programs</h1>
+      <div class="container d-flex my-3 justify-content-center line-bellow"></div>
+    </div>
+  </div>
+  <div class="ranking-section container-fluid py-5">
+    <h2 class="container text-cener pt-5 d-flex justify-content-center">Feature speakers</h2>
+    <div class="container d-flex my-3 justify-content-center cont-nav"></div>
```

Comment:
- [ ] The images on the featured speakers section are stretched out on the mobile view, please make sure to address also this issue.

![](https://i.imgur.com/BE39gmO.png)

Reply:
Addressed 📝